stdlib/http/client.pyi

```diff
@@ -142,6 +145,10 @@ class _HTTPConnectionProtocol(Protocol):
         ) -> HTTPConnection: ...
 
 class HTTPConnection:
+    auto_open: int = ...
```

Comment:
I think all of these except for `debuglevel` need to be marked `# undocumented` (see [here](https://github.com/python/typeshed/blob/master/CONTRIBUTING.md#what-to-include)).

Reply:
thanks, added it